packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -440,9 +440,10 @@ const makeDriver = async (t, initialPrice, priceBase) => {
     currentSeat: () => currentSeat,
     lastOfferResult: () => currentOfferResult,
     timer: () => timer,
-    tick: (ticks = 1) => {
+    tick: async (ticks = 1) => {
       for (let i = 0; i < ticks; i += 1) {
-        timer.tick();
+        // eslint-disable-next-line no-await-in-loop
+        await timer.tick();
```

Comment:
> This seems like a case where tick() doesn't need to be async, as long as all the callers await the results.

The callers can't await the result of `tick:` (line 443) unless it's async (as here) or returns a Promise. So yeah, this could instead be:
```
tick: (ticks = 1) => Promise.all(Array(ticks).map(() => timer.tick());
```
Do you think that'd be better?

Reply:
yes.